install/resources/mybb_theme.xml

```diff
@@ -12035,7 +12036,7 @@ if(use_xmlhttprequest == "1")
 					</tr>
 					<tr>
 						<td class="trow2" width="40%"><strong>{$lang->attachstats_spaceused}</strong></td>
-						<td class="trow2" width="60%">{$friendlyusage} ({$percent})</td>
+						<td class="trow2" width="60%">{$friendlyusage}</td>
```

Comment:
If removed completely, you can remove `attachments_usage_percent` too, because then there's only one reference to it (`attachments_usage_quota`) ☺️

Reply:
Modified `attachments_usage_percent` to hold required space and brackets.
Hard coded space removed from usercp.php
Done.